index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Why DOcumentin in Python</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="css/style.css">
+
+<style>
+ @media (max-width: 600px) {
+    body {
+        background-color: #f2f0eb;
+    }
+}
+</style>
+</head>
+<body>
+<main id="main-doc">
+ <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<h1>How and WHy Documenting in Python</h1>
+
+<div class="row">
+  <div class="col-lg-3 col-md-3">
+<nav id="navbar">
+				<header>
+				  Python Documentation
+				</header>
+
+				<ul>
+				  <li class="menu_item">
+					<a href="#introduction" class="nav-link">Introduction</a>
+				  </li>
+
+				  <li class="menu_item">
+					<a href="#why_documenting" class="nav-link">Why Documenting</a>
+				  </li>
+
+				  <li class="menu_item">
+					<a href="#commenting_vs_documenting" class="nav-link">Commenting Vs Documenting</a>
+				  </li>
+
+				  <li class="menu_item">
+					<a href="#commenting" class="nav-link">Commenting</a>
+				  </li>
+
+				  <li class="menu_item">
+					<a href="#documenting" class="nav-link">Documenting</a>
+				  </li>
+				</ul>
+			  </nav>
+
+  </div>
+  <div class="col-lg-9 col-md-9">
+
+  <!-- Section 1-->
+			  <section class="main-section" id="introduction">
+				<header>
+				  Introduction
+				</header>
+				<p>
+				  Welcome to your complete guide to documenting Python code. Whether you’re documenting a small script or a large project, whether you’re a beginner or a seasoned Pythonista, this guide will cover everything you need to know.
+				</p>
+			  </section>
+			  <!-- End of Section 1-->
+
+
+  </div>
+
+
+  <div class="col-lg-9 col-md-9">
+  <!-- section 2-->
+
+   <section class="main-section" id="why_documenting">
+				<header>
+				  Why Documenting
+				</header>
```

Comment:
Please remove the `<header> tag within the section `<section>. By doing so your document will comply with html5 semantic. Please consider replacing it with h1 ..h6 elements.
Instead of: `<section> <header> .....</header></section>` 
you should have: `<section><h1> ......</h1></section>`